specification/resources/databases/models/advanced_config/mysql_advanced_config.yml

```
@@ -254,3 +254,13 @@ properties:
     minimum: 1024
     maximum: 1048576
     example: 4096
+  log_output:
+    description: >-
+      Defines the destination for logs. Can be INSIGHTS, TABLE, or both (INSIGHTS,TABLE), or NONE to disable logs.
```

Looks like the order of items is important when we have multiple values - `INSIGHTS,TABLE` - as a "key": https://github.internal.digitalocean.com//digitalocean/cthulhu/blob/6cbc5a70bbef2702990933fabb6bbca9e14cfc2c/docode/src/do/teams/paas/dbaas/dbaas/aiven_provider.go#L6065-L6066 

Lets add some details about this case in a description:

```suggestion
      Defines the destination for logs. Can be `INSIGHTS`, `TABLE`, or both (`INSIGHTS,TABLE`), or `NONE` to disable logs. To specify both destinations, use `INSIGHTS,TABLE` (order matters).
```

---

@andrewsomething Do you happen to remember if we have any combined enum values like `INSIGHTS,TABLE` mentioned in the documentation? What's the best way to explain this clearly? 🙏